wavefront/resource_alert.go

```diff
@@ -194,7 +194,9 @@ func resourceAlertRead(d *schema.ResourceData, meta interface{}) error {
 	// Use the Wavefront ID as the Terraform ID
 	d.SetId(*tmpAlert.ID)
 	d.Set("name", tmpAlert.Name)
-	d.Set("target", tmpAlert.Target)
+	if tmpAlert.Target != "" && tmpAlert.AlertType == wavefront.AlertTypeClassic {
+		d.Set("target", tmpAlert.Target)
```

Comment:
What happens if this if statement doesn't trigger?  It appears the target field won't get set.